protocol/0068-MATC-matching_engine.md

```diff
@@ -16,7 +16,7 @@ Start date: 2021-12-14
        * Incoming [MARKET](./0014-ORDT-order_types.md#order-pricing-methods) MARKET orders will be matched fully if the volume is available, otherwise the order is cancelled. (<a name="0068-MATC-008" href="#0068-MATC-008">0068-MATC-008</a>) 
        * Incoming [LIMIT](./0014-ORDT-order_types.md#order-pricing-methods) orders will either be:
          * fully matched if possible to the other side of the book    (<a name="0068-MATC-009" href="#0068-MATC-009">0068-MATC-009</a>) 
-         * if a complete fill is not possible the order is cancelled without trading at all. (<a name="0068-MATC-010" href="#0068-MATC-010">0068-MATC-010</a>) 
+         * if a complete fill is not possible the order is stopped without trading at all. (<a name="0068-MATC-010" href="#0068-MATC-010">0068-MATC-010</a>) 
```

Comment:
what is the motivation to change from "cancelled" to "stopped"? On line 16, the order is also "cancelled".

Reply:
Because OSTA-001 says a **FOK** order which can't be filled is **stopped**, not cancelled.